modules/encrypt/config/encrypt.php

```diff
@@ -2,20 +2,20 @@
 /**
  * The following options must be set:
  *
- * string   key     secret passphrase
+ * string   type    Driver Type
+ * string   key     Secret Passphrase
  */
-
 return [
-    'default' => [
-        Kohana_Encrypt_Engine::CONFIG_TYPE => Kohana_Encrypt_Engine_Openssl::TYPE,
-        Kohana_Encrypt_Engine::CONFIG_KEY => NULL,
-        //Additional OpenSSL configuration
-        Kohana_Encrypt_Engine::CONFIG_CIPHER => Kohana_Encrypt_Engine_Openssl::AES_256_CBC,
-    ],
-//    'sodium' => [
-//        Kohana_Encrypt_Engine::CONFIG_TYPE => Kohana_Encrypt_Engine_Sodium::TYPE,
-//        Kohana_Encrypt_Engine::CONFIG_KEY => NULL,
-//    ],
+	'default' => [
+		'type'    => 'sodium',
```

Comment:
Are we sure sodium is the new default and not openssl?

Reply:
Actually i don't care about that i just tested and my commit wasn't "final" but yeah let's switch it to openssl :)